### ISSUE TITLE
.torrent file

### ISSUE THREAD

**Issue by navneet08** [patheticGeek/torrent-aio-bot#48]
Well , telegram bot is not responding .
But the site is working good , one problem is that one can't add .torrent files .
Please add the feature to add torrent with its .torrent file .

**Comment by askfriends** [comment]
yes, i also would love to have this. As most of the private trackers dont allow magnet url

**Comment by askfriends** [comment]
any update on this???

**Comment by navneet08** [comment]
> any update on this???

don't think so !
use "maple31" repo on aria2c web interface -- ariaNg .
it's awesome to mirror torrents into your google drive :)

**Comment by askfriends** [comment]
> > any update on this???
> 
> don't think so !
> use "maple31" repo on aria2c web interface -- ariaNg .
> it's awesome to mirror torrents into your google drive :)

yes i am already using that, but i want a telegram bot which does the same job

**Comment by vaibhavdost** [comment]
> > any update on this???
> 
> don't think so !
> use "maple31" repo on aria2c web interface -- ariaNg .
> it's awesome to mirror torrents into your google drive :)

Heroku suspended my two accounts
